src/main/java/com/redhat/coolstore/service/CatalogService.java

```diff
@@ -1,20 +1,21 @@
 package com.redhat.coolstore.service;
 
 import java.util.List;
-import java.util.logging.Logger;
+import org.jboss.logging.Logger;
 
 import javax.inject.Inject;
 
+import javax.enterprise.context.ApplicationScoped;
+
 import javax.persistence.criteria.CriteriaBuilder;
 import javax.persistence.criteria.CriteriaQuery;
 import javax.persistence.criteria.Root;
 
-import javax.ejb.Stateless;
 import javax.persistence.EntityManager;
 
 import com.redhat.coolstore.model.*;
 
-@Stateless
+@ApplicationScoped
```

Comment:
This is correct, but it's missing the _javax to jakarta_ namespace changes. Probably because we're seeing only one rule being applied.